Q/Qt/build_tarballs.jl

```diff
@@ -0,0 +1,154 @@
+        
+        cd $WORKSPACE/srcdir/build
+        apk add g++ linux-headers
+        export PATH=$(echo "$PATH" | sed -e 's/\/opt\/bin://')
+        export QT_MAC_SDK_NO_VERSION_CHECK=1
+        ../qt-everywhere-src-5.14.1/configure \
+            QMAKE_CXXFLAGS+=-F/opt/$target/$target/sys-root/System/Library/Frameworks \
+            QMAKE_RANLIB=/opt/bin/ranlib \
+            QMAKE_MACOSX_DEPLOYMENT_TARGET=10.14 \
+            -platform linux-g++ -xplatform macx-clang -device-option CROSS_COMPILE=/opt/bin/$target- \
+            -prefix $prefix $commonoptions \
+            -skip qtwinextras
+        ;;
+        
+	*w64-mingw*)
+        apk add g++ linux-headers
+        export PATH=$(echo "$PATH" | sed -e 's/\/opt\/bin://')
+        
+        ../qt-everywhere-src-5.14.1/configure -platform linux-g++ -xplatform win32-g++ -device-option CROSS_COMPILE=/opt/bin/$target- \
+            -prefix $prefix $commonoptions \
+            -opengl desktop
+		;;
+
+    *arm-linux*)
+        apk add g++ linux-headers
+        export PATH=$(echo "$PATH" | sed -e 's/\/opt\/bin://')
+        export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$prefix/share/pkgconfig
+        export PKG_CONFIG_LIBDIR=$prefix/lib/pkgconfig
+        
+        ../qt-everywhere-src-5.14.1/configure QMAKE_LFLAGS=-liconv -platform linux-g++ -device linux-rasp-pi3-g++ -device-option CROSS_COMPILE=/opt/bin/$target- \
+            -extprefix $prefix $commonoptions \
+            -skip qtwinextras -system-xcb -fontconfig -sysroot /opt/$target/bin/../$target/sys-root
+		;;
+esac
+
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:armv7l, libc=:glibc, call_abi=:eabihf),
+    Windows(:x86_64),
+    Linux(:x86_64, libc=:glibc)
+]
+platforms = expand_cxxstring_abis(platforms)
+
+platforms_macos = [ MacOS(:x86_64) ]
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct(["Qt5Core", "libQt5Core", "QtCore"], :libqt5core),
+    LibraryProduct(["Qt5Quick", "libQt5Quick", "QtQuick"], :libqt5quick),
+    LibraryProduct(["Qt5QuickControls2", "libQt5QuickControls2", "QtQuickControls2"], :libqt5quickcontrols2),
+    LibraryProduct(["Qt5Svg", "libQt5Svg", "QtSvg"], :libqt5svg),
+    LibraryProduct(["Qt5Widgets", "libQt5Widgets", "QtWidgets"], :libqt5widgets),
+]
+
+products_macos = [
+    FrameworkProduct("QtCore", :libqt5core),
+    FrameworkProduct("QtQuick", :libqt5quick),
+    FrameworkProduct("QtQuickControls2", :libqt5quickcontrols2),
+    FrameworkProduct("QtSvg", :libqt5svg),
+    FrameworkProduct("QtWidgets", :libqt5widgets),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency("Xorg_libX11_jll"),
+    Dependency("Xorg_libXext_jll"),
+    Dependency("Xorg_glproto_jll"),
```

Comment:
Do you really need this package?  Even if you do, I assume you don't need it at runtime -- it's only a collection of header files -- so this can probably become
```suggestion
    BuildDependency("Xorg_glproto_jll"),
```

Reply:
Yes, I didn't realise this was implemented, cool!